backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxIteratorApi.scala

```diff
@@ -94,14 +93,14 @@ class VeloxIteratorApi extends IIteratorApi with Logging {
    * @return
    */
   override def genCoalesceIterator(
-      iter: Iterator[ColumnarBatch],
-      recordsPerBatch: Int,
-      numOutputRows: SQLMetric,
-      numInputBatches: SQLMetric,
-      numOutputBatches: SQLMetric,
-      collectTime: SQLMetric,
-      concatTime: SQLMetric,
-      avgCoalescedNumRows: SQLMetric): Iterator[ColumnarBatch] = {
+                                    iter: Iterator[ColumnarBatch],
```

Comment:
This might be subjective but this kind of alignment is often a waste of code space. Is this from the style config file?

Reply:
yes, the line length can only be 100